types/valuerenderer/valuerenderer_test.go

```diff
@@ -0,0 +1,168 @@
+package valuerenderer_test
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/valuerenderer"
+)
+
+// TODO add more test cases
+func TestFormatCoin(t *testing.T) {
```

Comment:
I am thinking to add more  test cases here e.g.  'mregen` -> `uregen`, `uregen` -> `regen` etc